src/wp-content/themes/twentytwelve/header.php

```diff
@@ -33,6 +33,7 @@
 
 <body <?php body_class(); ?>>
 <?php wp_body_open(); ?>
+<a class="assistive-text" href="#content"><?php _e( 'Skip to content', 'twentytwelve' ); ?></a>
 <div id="page" class="hfeed site">
 	<header id="masthead" class="site-header">
```

Comment:
```suggestion
<div id="page" class="hfeed site">
	<a class="screen-reader-text skip-link" href="#content"><?php _e( 'Skip to content', 'twentytwelve' ); ?></a>
	<header id="masthead" class="site-header">
```

1. I prefer placing the link inside the "page" `div` (as Twenty Fifteen and some later bundled themes do). If someone uses a link to `#page`, then the skip link would be the first item.
2. If you also change the `assistive-text` class to `screen-reader-text`, Twenty Twelve already has styles for `.site .screen-reader-text:focus`. Then `style.css` would not need any editing.